src/clarity-angular/popover/popover.ts

```diff
@@ -285,25 +277,11 @@ export class Popover {
         }
     }
 
-    private resumeScrolling() {
-        for (let container of this.originalOverflows) {
-            this.setInlineOverflow(container.e, container.overflow);
+    private removeScrollEventListeners() {
+        for (let elem of this.scrollableElements) {
+            elem.removeEventListener("scroll");
```

Comment:
You should have a single listener function which you specifically remove by calling `elem.removeEventListener("scroll", listener);`. Because right now, you might remove an scroll listener that's not ours. See https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Memory_issues (which means that on top of safely removing only our listener, we save some memory).